Makefile.am

```diff
@@ -123,10 +111,14 @@ if USE_CRYPTOPP
 endif
 if BUILD_CLIENT
 	rm -f $(DESTDIR)$(prefix)/lib/libCppBlockUtils.*
+if BUILD_GUI
 	install_name_tool -change $(DESTDIR)$(prefix)/lib/libArmoryGUI.0.dylib @loader_path/../libArmoryGUI.0.dylib $(DESTDIR)$(prefix)/lib/armory/_CppBlockUtils.so
 	install_name_tool -change $(DESTDIR)$(prefix)/lib/libArmoryCommon.0.dylib @loader_path/libArmoryCommon.0.dylib $(DESTDIR)$(prefix)/lib/libArmoryGUI.0.dylib
+endif
 if USE_CRYPTOPP
+if BUILD_GUI
 	install_name_tool -change $(DESTDIR)$(prefix)/lib/libcryptopp.0.dylib @loader_path/libcryptopp.0.dylib $(DESTDIR)$(prefix)/lib/libArmoryGUI.0.dylib
+endif
```

Comment:
Shouldn't this be extended to cover the _CppBlockUtils.so file below?

Reply:
I haven't gotten around to the GUI part.

Reply:
I figured we should probably hold off on that until after the cffi conversions.

Reply:
That's fine. Just wanted to understand. :)